frontend/src/Store/Actions/movieIndexActions.js

```diff
@@ -223,16 +223,28 @@ export const defaultState = {
       type: filterBuilderTypes.EXACT,
       valueType: filterBuilderValueTypes.BOOL
     },
+    {
+      name: 'isAvailable',
+      label: translate('Is Available'),
```

Comment:
Translate keys wont have spaces in them

`src\NzbDrone.Core\Localization\Core\en.json` is where you would check for existing keys or add new ones as needed